### ISSUE TITLE
Accuracy: Add reset button

### ISSUE THREAD

**Issue by gdt** [barbeau/gpstest#533]
I was standing someplace, started GPSTest, ran it for a while to watch the satellite count go up, and then put it on a (nearby but not exactly) mark and started accuracy.   It seemed to have samples from before, and there was no obvious way to clear them to only average the accuracy from after my desired start time.

I'd like two things.  One is a reset button that after it is pushed, data from before the push is not included in the stats.
The second thing is that when accuracy mode is entered, an implicit reset should happen.  There's no reason to believe that the user was in position before invoking accuracy mode, and I at least don't expect the past to be included.

**Comment by vkbellis** [comment]
The accuracy counter will automatically reset to zero by simply tapping on the edit button without actually editing your known location's value.

![accuracy 20210625 092430](https://user-images.githubusercontent.com/82275968/123432252-2eb08e00-d598-11eb-85ff-5014f94b50f6.PNG)

**Comment by barbeau** [comment]
@gdt Yes, for now a workaround is just to tap "Edit" and then "Save" again for your ground truth location, and it will reset the counter back to 0. I agree more explicit controls would be useful here, and I'll leave this issue open for future improvements.

**Comment by gdt** [comment]
Hey @stale  this is still valid.

**Comment by gdt** [comment]
Hey https://github.com/Stale this is still valid.

**Comment by gdt** [comment]
Hey https://github.com/Stale this is still valid.

**Comment by barbeau** [comment]
Not stale
